content/tracing/setup/java.md

```diff
@@ -255,6 +255,56 @@ public class MyClass {
 }
 ```
 
+## Logging and MDC
+
+The Java tracer exposes two api calls to allow printing trace and span identifiers along with log statements, `CorrelationIdentifier#getTraceId()` and `CorrelationIdentifier#getSpanId()`.
```

Comment:
capitalize API, use the serial comma: 
The Java tracer exposes two API calls to allow printing trace and span identifiers along with log statements, `CorrelationIdentifier#getTraceId()`, and `CorrelationIdentifier#getSpanId()`.

Reply:
👍 Changes pushed.